check_logs.py

```diff
@@ -1031,6 +1031,22 @@ def print_ansible_errors(args, errors):
         sh.values_update(
             worksheet.title, {"valueInputOption": "USER_ENTERED"}, {"values": values}
         )
+    if args.github_action_format:
+        for error in errors:
```

Comment:
I'm a bit confused by this function -- It's called `print_ansible_errors`, but it creates a spreadsheet. With `--gh-format` should still actually do that, or is it more of an if/else thing? To my knowledge we don't put any spread sheet file into the test artifact.

Reply:
The output format options listed in `print_ansible_errors` are not mutually exclusive - you could output all of CSV, a google spreadsheet, and github action format at the same time.  That's probably unlikely.

Reply:
Note that `print_ansible_errors` will not output a google spreadsheet unless you pass in the `--gspread` and other related arguments.

Reply:
Ack, thanks!